actions/class.Runner.php

```diff
@@ -697,10 +698,17 @@ public function timeout()
             $this->validateSecurityToken();
             $this->validateDeliveryExecutionInteractionAccessibility();
 
+            $container = $this->getServiceLocator()->getContainer();
+            $isPermanentLateSubmission = $container->has(FeatureFlagChecker::class)
+                ? $container->get(FeatureFlagChecker::class)
+                            ->isFeatureEnabled(TimeoutCommand::FEATURE_FLAG_TIMEOUT_PERMANENT_LATE_SUBMISSION)
+                : false;
```

Comment:
Do we need this? Feature flag checker is a part of tao-core and must be registered in container. 🤔 
Also, instead of
```php
$this->getServiceLocator()->getContainer();
```
you can use
```php
$this->getPsrContainer();
```